codalab/server/worker_info_accessor.py

```diff
@@ -1,13 +1,20 @@
 from collections import defaultdict
 import datetime
+from typing import Callable, Any, Optional, Union
 
 
 class WorkerInfoAccessor(object):
     """
     Helps with accessing the list of workers returned by the worker model.
     """
 
-    def refresh_cache(f):
+    def refresh_cache(
+        f: Union[
+            Callable[['WorkerInfoAccessor'], Any],
```

Comment:
Can we do something about `Any` here?

Reply:
Sometimes it returns a list, sometimes a boolean and sometimes even nothing. I feel like making it as Any fits the purpose of this decorator better, because it's used to pre-process parameters. I can change the parameters to str, as it's the only type showed up.

Reply:
I can change the return type to what we have for now to see if it passes tests.